prime-router/build.gradle.kts

```diff
@@ -343,7 +343,6 @@ tasks.azureFunctionsRun {
             "POSTGRES_PASSWORD" to dbPassword,
             "POSTGRES_URL" to dbUrl,
             "PRIME_ENVIRONMENT" to "local",
-            "REDOX_SECRET" to "some_secret",
```

Comment:
Note that just yanking this does not obviate the need to suppress the warning; the nightly run checks the history, not just the state of master as is. In other words, unless you supress this one, the nightly gitleaks run will continue to flag this.